pkg/discovery/mdns.go

```diff
@@ -40,7 +42,7 @@ func RegisterMDNS(id string, port int) error {
 func LookupPortMDNS(id string) (int, error) {
 	resolver, err := zeroconf.NewResolver(nil)
 	if err != nil {
-		log.Fatalln("Failed to initialize resolver:", err.Error())
+		log.Fatalf("Failed to initialize resolver:", err.Error())
```

Comment:
Does this behave the same way as before? I expect that Fatalf() would refer to the arguments passed in while the original version would simply append the second argument to the output.

Reply:
nice catch. I need to add %v

Reply:
fixed.